<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchPopup</class>
 <widget class="QWidget" name="SearchPopup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>66</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>3</number>
   </property>
   <item row="0" column="1">
    <widget class="QLabel" name="findL">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Find:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="SearchComboBox" name="findCmb">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QComboBox" name="replaceCmb">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="closeBtn">
     <property name="maximumSize">
      <size>
       <width>26</width>
       <height>26</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="7">
    <widget class="QPushButton" name="replaceAllBtn">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>26</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Replace all</string>
     </property>
     <property name="text">
      <string>*</string>
     </property>
    </widget>
   </item>
   <item row="2" column="8" colspan="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="4">
    <widget class="QPushButton" name="findNextBtn">
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>26</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Find next</string>
     </property>
     <property name="text">
      <string>v</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QPushButton" name="replaceNextBtn">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>26</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Replace next</string>
     </property>
     <property name="text">
      <string>v</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="findPrevBtn">
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>26</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Find previous</string>
     </property>
     <property name="text">
      <string>^</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="replacePrevBtn">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>26</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Replace previous</string>
     </property>
     <property name="text">
      <string>^</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="replaceL">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Replace:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="10">
    <widget class="QComboBox" name="modeCmb">
     <item>
      <property name="text">
       <string>Plain text</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Whole words</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Regexp</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Multi-line regexp</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="8">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="9">
    <widget class="QCheckBox" name="caseSensitiveChk">
     <property name="text">
      <string>Case sensitive</string>
     </property>
    </widget>
   </item>
   <item row="0" column="7">
    <widget class="QPushButton" name="expandBtn">
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>26</height>
      </size>
     </property>
     <property name="text">
      <string>.</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SearchComboBox</class>
   <extends>QComboBox</extends>
   <header>SearchComboBox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>closeBtn</tabstop>
  <tabstop>findCmb</tabstop>
  <tabstop>findPrevBtn</tabstop>
  <tabstop>findNextBtn</tabstop>
  <tabstop>replaceCmb</tabstop>
  <tabstop>replacePrevBtn</tabstop>
  <tabstop>replaceNextBtn</tabstop>
  <tabstop>replaceAllBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
